pkgs/development/python-modules/yowsup/default.nix

```diff
@@ -1,10 +1,14 @@
-{ buildPythonPackage, stdenv, fetchFromGitHub, six, python-axolotl, pytest }:
+{ buildPythonPackage, stdenv, fetchFromGitHub, six, python-axolotl, pytest
+, disabled
```

Comment:
don't add it as parameter. Just add it directly to the call to `buildPythonPackage`.